packages/wdio-spec-reporter/tests/__snapshots__/index.test.ts.snap

```diff
@@ -157,9 +157,13 @@ exports[`SpecReporter > onRetry > should group retried test cases 1`] = `
 [loremipsum 50 Windows 10 #0-0] Foo test
 [loremipsum 50 Windows 10 #0-0]    green ✓ foo
 [loremipsum 50 Windows 10 #0-0]    green ✓ bar
+[loremipsum 50 Windows 10 #0-0]    red ✖ loo
 [loremipsum 50 Windows 10 #0-0]    green ✓ loo (1 retries)
```

Comment:
This is not desired. If a test gets retried the failing tests before should not be displayed.

Reply:
The retries come from specFileRetries in wdio config. Though it is initialised as 0 in test:start, while test ends, it is not updated but rather updated 'retries' is passed to onRunnerEnd which is not updating the retries in hooksAndTests. I ran with old original code and still (1 retries) is not getting printed. I tried with both specFileRetries and retry under cucumberOpts.